test/dart3_suggestors/null_safety_prep/required_flux_props_test.dart

```diff
@@ -0,0 +1,1160 @@
+// Copyright 2023 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:meta/meta.dart';
+import 'package:over_react_codemod/src/dart3_suggestors/null_safety_prep/required_flux_props.dart';
+import 'package:test/test.dart';
+
+import '../../resolved_file_context.dart';
+import '../../util.dart';
+import '../../util/component_usage_migrator_test.dart';
+
+void main() {
+  final resolvedContext = SharedAnalysisContext.overReact;
+
+  // Warm up analysis in a setUpAll so that if getting the resolved AST times out
+  // (which is more common for the WSD context), it fails here instead of failing the first test.
+  setUpAll(resolvedContext.warmUpAnalysis);
+
+  group('RequiredFluxProps', () {
+    late SuggestorTester testSuggestor;
+
+    setUp(() {
+      testSuggestor = getSuggestorTester(
+        RequiredFluxProps(),
+        resolvedContext: resolvedContext,
+      );
+    });
+
+    test(
+        'leaves builders alone if they don\'t use FluxUiPropsMixin, '
+        'even if they have props named store/actions', () async {
+      await testSuggestor(
+        isExpectedError: (err) =>
+            err.message.contains(RegExp(r'theStore|theActions')),
```

Comment:
Is this error to ignore `unused_local_variable`? Could we test for that error message instead, to help avoid accidentally ignoring other errors related to those variables that could be indicative of a bad test setup?

Reply:
Addressed in https://github.com/Workiva/over_react_codemod/pull/268/commits/6a3aafc1a64e15b27034b8f6221bcab2ff4525f8